spoonbill/stats.py

```diff
@@ -224,7 +226,7 @@ def process_items(self, releases, with_preview=True):
                     elif isinstance(item, list):
                         abs_pointer = separator.join([abs_path, key])
                         if not isinstance(item[0], dict) and not item_type:
-                            LOGGER.warning(
+                            LOGGER.debug(
```

Comment:
I think, when reducing log level to DEBUG we need to replace string.format with % operator.